cnd/src/http_api/routes.rs

```diff
@@ -173,27 +174,18 @@ where
             );
             entity.push_sub_entity(beta_state_sub);
 
-            let maybe_action_names = vec![
-                swap.init_action().map(|_| ActionName::Init),
-                swap.deploy_action().map(|_| ActionName::Deploy),
-                swap.fund_action().map(|_| ActionName::Fund),
-                swap.redeem_action().map(|_| ActionName::Redeem),
-                swap.refund_action().map(|_| ActionName::Refund),
```

Comment:
By removing this, you are removing the **intended** coupling between _can we construct the action_ and _should we tell the user that the action is available_.

The code between `NextRecommendedAction` and `XYZAction` can now potentially run out of sync which would result in the user seeing an action on the swap but when they actually hit the endpoint, it returns 404.